spaces/S000043/properties/P000149.md

```diff
@@ -0,0 +1,10 @@
+---
+space: S000043
+property: P000149
+value: false
+refs:
+  - mr: MR3728284
+    name: Topology, 2nd Edition (Munkres)
+---
+
+As proved on page 193 of {{mr:MR3728284}}, the {S76} is not {P18}, so the Sorgenfrey line is not Lindelöf in all of its finite powers.
```

Comment:
For S43/P149, the justification that S76 is P18 does not need to be justified by Munkres, I would say, as the user can just click S76 and check it there.  If we need a token something in the refs list, maybe just put Countexamples there?

Reply:
Yes, I wanted to simply reference the property for the space, but wasn't sure if there was a way to link directly to the property for the space, particularly. I included Munkres to be on the safe side in terms of references, even if it is a bit overkill. I will be happy to update this. For S76/P18, we currently mention that the antidiagonal is closed and discrete and mention that every closed subset of a Lindelof space is Lindelof. That's effectively what's in Munkres, as well, and I would imagine what's in Counterexamples.

Reply:
created https://github.com/pi-base/web/issues/66